sdks/python/apache_beam/examples/cookbook/multiple_output_pardo_test.py

```diff
@@ -18,54 +18,43 @@
 """Test for the multiple_output_pardo example."""
 
 import logging
-import re
-import tempfile
 import unittest
 
+import apache_beam as beam
 from apache_beam.examples.cookbook import multiple_output_pardo
+#from apache_beam.transforms.util import assert_that
+#from apache_beam.transforms.util import DataflowAssertException
 
 
-class MultipleOutputParDo(unittest.TestCase):
+class MultipleOutputParDoTest(unittest.TestCase):
 
-  SAMPLE_TEXT = 'A whole new world\nA new fantastic point of view'
-  EXPECTED_SHORT_WORDS = [('A', 2), ('new', 2), ('of', 1)]
-  EXPECTED_WORDS = [
-      ('whole', 1), ('world', 1), ('fantastic', 1), ('point', 1), ('view', 1)]
-
-  def create_temp_file(self, contents):
-    with tempfile.NamedTemporaryFile(delete=False) as f:
-      f.write(contents)
-      return f.name
-
-  def get_wordcount_results(self, temp_path):
-    results = []
-    with open(temp_path) as result_file:
-      for line in result_file:
-        match = re.search(r'([A-Za-z]+): ([0-9]+)', line)
-        if match is not None:
-          results.append((match.group(1), int(match.group(2))))
-    return results
+  SAMPLE_TEXT = 'A whole new world\nA new point'
+  text_len = len(' '.join(SAMPLE_TEXT.split('\n')))
+  SAMPLE_TEXT_Iterable = [SAMPLE_TEXT]
+  EXPECTED_SHORT_WORDS = ['A: 2', 'new: 2']
+  EXP_WORDS = ['point: 1', 'whole: 1', 'world: 1']
```

Comment:
Variable naming is not consistent within this file and with other tests. Could you fix that, for example:

`EXP_WORDS` -> `EXPECTED_WORDS`
`SAMPLE_TEXT_Iterable` -> `SAMPLE_TEXT_ITERABLE`
`res _sh_wrd` -> `res_short_words`
`res_cnt` -> `res_count`

Maybe even spell out `res` to `results`.